src/translations.h

```diff
@@ -16,7 +16,13 @@
 #include <libintl.h>
 #include <clocale>
 
-#define _(STRING) gettext(STRING)
+extern char *blank_string;
+
+inline char *_( const char *msg )
+{
+    return ( msg[0] == '\0' ) ? blank_string : gettext( msg );
```

Comment:
> Casting  const char*  to  char*  is prohibited.

Yes, that's why the function should return a `const char *`. The dummy macro that is used when `LOCALIZE` is not defined (see a few lines below) just forwards the input, which is already a `const char *` (it "returns" `const char *`) and quote from https://www.gnu.org/software/libc/manual/html_node/Translation-with-gettext.html

> Please note that although the return value is char * the returned string must not be changed. This broken type results from the history of the function and does not reflect the way the function should be used.

The result of `gettext` is effectively a `const char *`.